src/utilities/activity-utils.ts

```diff
@@ -2,6 +2,7 @@ import { Page, Activity, EmbeddableWrapper } from "../types";
 import { SidebarConfiguration } from "../components/page-sidebar/sidebar-wrapper";
 import { isQuestion as isEmbeddableQuestion } from "./embeddable-utils";
 import { runningInCypress } from "./cypress";
+import { isGenerator } from "mobx/dist/internal";
```

Comment:
Looks like this import isn't being used.

Reply:
Thanks for spotting this.  I am not sure how that got inserted, possibly some eager VS Code autocomplete. Its removed in d23f524